README.md

```diff
@@ -52,115 +56,48 @@ With Go:
-previous experience working with large-scale platforms in some of the most sophisticated companies in the world.
-It focuses on significantly improving the developer experience and productivity and integrates with existing tooling
-in a non-intrusive way without major lock-ins and for a broader audience.
-
-If you like Terramate CLI, please give the project a ⭐ and join our [Discord Community](https://discord.gg/CyzcScEPkc).
-
-## What are the most common use cases of Terramate CLI?
-
-📈 **Implement and maintain highly scalable IaC projects**
-
-Stacks unlock faster execution, better collaboration, and lower blast radius risk. With stacks, parallel deployment
-ensures no more pipeline run bottlenecks.
-
-🔗 **GitOps workflows that run in your existing CI/CD platform**
+## Getting Started
 
-Unlock CI/CD workflows to orchestrate your stacks in your existing CI/CD platform, such as GitHub Actions, GitLab and
-Bitbucket Pipelines. Can be used to **preview changes and plans in Pull Requests** or to **detect infrastructure drift**
-with scheduled runs.
+The fastest way to get started with Terramate is our [getting started guide](https://terramate.io/docs/cli/getting-started/).
 
-🏎 **Deploy significantly faster and more often**
+## Why simply complex codebases?
 
-Change detection allows you to execute only commands in stacks that have been changed in the current branch or since
-the last merge, reducing CI/CD runtimes to a bare minimum, even in complex environments, **saving costs** and
-**improving throughput.**
+As Terraform and OpenTofu offer no standard pattern on how to organize code efficiently, projects quickly sprawl out of
+control. The consequences are ***code complexity***, ***long-running pipelines***, ***poor governance***,
+***drifts***, and ***high blast radius***.
 
-👷 **Better developer productivity and experience**
+Terramate CLI enables a new approach to eliminating the root cause of these consequences: using a modular and efficient
+approach called Terramate Stacks. A stack is a combination of infrastructure code, state and configuration, which can be
+nested to split state further.
 
-Simplified management and improved productivity by providing developer-centric workflows like one-click cloning stacks
-and environments. Enable developers to use templates to deploy complex infrastructure without requiring them to
-learn the complexity of IaC.
+Organisationally, a stack is a way to establish an ownership model. By consistently sizing stacks into isolated units
+that your team members can quickly understand, assuming ownership of your infra becomes a lot simpler.
+Terramate CLI supports Terraform, OpenTofu and Terragrunt, and can be run locally as well as in your existing VCS
+(Github, Gitlab) CI/CD.
 
-⚒️ **Data Sharing**
+Terramate CLI is designed to work especially well with [Terramate Cloud](https://cloud.terramate.io), which provides
+an IaC management layer and offers a generous free tier. 
```

Comment:
This doesn't sound obviously great. Maybe we can list some of the benefits of the cloud here? I understand we want to keep it subtle.